lib/screens/container/driver_list.dart

```diff
@@ -54,7 +90,18 @@ class _DriverContainerListState extends State<DriverContainerList> {
               return Column(
                 children: [
                   ExpansionTile(
-                    leading: Icon(Icons.home_work),
+                    leading: Column(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: [
+                        Icon(Icons.home_work),
+                        Text(
+                          t.size == null ? 'N 20 s t' : t.size.text,
```

Comment:
The default value should not be some hard coded string but a .text on the default ContainerSize enum